x-pack/plugins/grokdebugger/public/lib/ace/grok_highlight_rules.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import ace from 'ace';
+
+const TextHighlightRules = ace.acequire("ace/mode/text_highlight_rules").TextHighlightRules;
+
+export class GrokHighlightRules extends TextHighlightRules {
+  constructor() {
+    super();
+    this.$rules = this.getHighlightRules();
```

Comment:
Why is this value assigned via method call instead of a constant?

Reply:
You're right, there's no reason for it to be a method. Changing...